src/scripts.js

```diff
@@ -10,204 +10,145 @@
 // import User from './user';
 // import Cookbook from './cookbook';
 // import domUpdates from './domUpdates';
-
+let favorites = []
+let recipeData;
 let favButton = document.querySelector('.view-favorites');
 let homeButton = document.querySelector('.home')
 let cardArea = document.querySelector('.all-cards');
 const recipeCards = document.querySelector('.all-cards');
 let pantryArea = document.querySelector('.pantry-cards');
 // let cookbook = new Cookbook(recipeData);
+favButton.addEventListener('click', ()=>{
+  if(!favButton.classList.contains('clicked')){
+  domUpdates.displayFavorites(findFavorites(favorites,recipeData))
+  favButton.classList.add('clicked')
+  }
+  else{
+    favButton.classList.remove('clicked')
+    displayAllRecipes(recipeData)
+  }
+})
 let user, pantry;
 
+
+recipeCards.addEventListener('click', () => {
+if(event.target.classList.contains('star-icon')){
+  toogleFavorites(event)
+}
+})
+function toogleFavorites(event){
+  if(event.target.src === "https://image.flaticon.com/icons/svg/149/149222.svg"){
+    event.target.src =   "https://image.flaticon.com/icons/svg/148/148841.svg"
+    addToFavorites(event)
```

Comment:
Same thing here, there is an `addToFavorites` method on the `User` object for this :)